gitautosync/util.py

```diff
@@ -0,0 +1,6 @@
+import logging
+
+logging.basicConfig(
```

Comment:
Not sure. I think it's just a formatting preference. I copied it over from nbinteract.